### PR TITLE
Add versioning manual

### PR DESCRIPTION
**Pull request by ybznek**
Add versioning manual

Waiting for https://github.com/OpenSCAP/website/issues/84

Do not merge it now.

### CODE REVIEW

**Comment by mpreisler** [comment]
Why `maint-1.2`? This needs to go into `maint-1.0` IMO.

**Comment by msrubar** [comment]
I'm confused about the distribution. I thought that the original idea was just to put this document from the old website to the asciidoc format into the contribute folder. Why do we want the html version of it? Do we want to have in on the static page?

**Comment by mpreisler** [comment]
That's a good point, @msrubar. Consumers of the tarball most likely don't care about our versioning policy. Does it make sense for us to put this document into the tarball?

**Comment by ybznek** [comment]
I've opened pull request to make some place to discusion, so thanks for your feedback.

@mpreisler We was talking about reference from README.md and it is only in maint-1.2.

So, should I move it to maint-1.0 and don't compile to html?

File: docs/contribute/versioning.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Bugfixes applied to maintenance branches must be merged up. For example a bugfix applied to maint-1.0 shall be merged to maint-1.2 which then shall be merged to master.
+
+This rule makes sure we always carry bugfixes to new feature releases.
+
+In cases where the bugfix does not make sense in newer feature releases, a noop merge shall be done. For example a fix for functionality that was removed in newer feature release.
+
+== Maintenance status
+
+	* 1.2.x: supported
+	* 1.1.x: unsupported
```
Comment (msrubar) [inline docs/contribute/versioning.adoc]: maybe a note that it'll be removed soon?

Comment (mpreisler) [inline docs/contribute/versioning.adoc]: please change this to "1.1.x: unsupported since 2015-10-16."

**Comment by ybznek** [comment]
Replaced by https://github.com/OpenSCAP/openscap/pull/185

**Pull request closed without merging**